network-api/networkapi/mozfest/templates/mozfest/mozfest-base.html

```diff
@@ -27,18 +31,24 @@
 
 {% block content %}
   {% block secondary_nav %}{% endblock %}
+  <div class="container">
+    <div class="row my-1">
```

Comment:
nitpicking: if we wanna match the layout we have on [this page](https://foundation.mozilla.org/en/mozfest-save-the-date/), then we should remove `my-1` from here as it adds some extra vertical margin.

Reply:
```suggestion
    <div class="row">
```